packages/client-core/src/common/services/ProjectService.ts

```diff
@@ -77,33 +77,37 @@ export const ProjectState = defineState({
 
 //Service
 export const ProjectService = {
-  fetchProjects: async () => {
+  fetchProjects: () => {
     try {
-      const projects = (await Engine.instance.api.service(projectPath).find({
+      const projectsQuery = useFind(projectPath, {
         query: {
           action: 'admin',
           allowed: true
         }
-      })) as Paginated<ProjectType>
+      })
+
       getMutableState(ProjectState).merge({
         updateNeeded: false,
-        projects: projects.data
+        projects: projectsQuery.data
       })
     } catch (err) {
       NotificationService.dispatchNotify(err.message || JSON.stringify(err), { variant: 'error' })
     }
   },
 
   // restricted to admin scope
-  createProject: async (name: string, params?: ProjectUpdateParams) => {
-    const result = await Engine.instance.api.service(projectPath).create({ name }, params)
-    logger.info({ result }, 'Create project result')
-    await ProjectService.fetchProjects()
+  createProject: (name: string, params?: ProjectUpdateParams) => {
+    const result = useService(projectPath, 'create', {
```

Comment:
these need to all stay with the old API, react hooks can only be used in a react context